extension/js/common/helpers.ts

```diff
@@ -164,3 +164,11 @@ export const processAndStoreKeysFromEkmLocally = async (
   }
   return { updateCount: encryptedKeys?.keys.length ?? 0 + (existingKeys.length - keysToRetain.length), noKeysSetup: !(encryptedKeys?.keys.length || keysToRetain.length) };
 };
+
+export const getLocalKeyExpiration = async ({ acctEmail }: Bm.GetLocalKeyExpiration): Promise<Bm.Res.GetLocalKeyExpiration> => {
+  const keys = await KeyStore.get(acctEmail);
+  const keyInfos = await Promise.all(keys.map(async (key) => await KeyUtil.parse(key.public)));
```

Comment:
We often name the other way around. `KeyStore.get` returns KeyInfo's (kis) -- armored keys with some additional info.
And parsed keys can be named `parsedKeys` or simply `keys`